code/modules/admin/topic.dm

```diff
@@ -2071,6 +2071,53 @@
 
 		usr << browse(dat.Join("<br>"), "window=related_[C];size=420x300")
 
+	else if(href_list["centcomlookup"])
+		if(!check_rights(R_ADMIN))
+			return
+
+		var/ckey = href_list["centcomlookup"]
+
+		var/list/query = json_decode(rustg_http_request_blocking(RUSTG_HTTP_METHOD_GET, "https://centcom.melonmesa.com/ban/search/[ckey]", null, null))
```

Comment:
This must be configurable, and the server behind this must be open source, and the system must support pushing and pulling from multiple db backends incase everybody just wants to setup their own server and or other trust issues arise.

Reply:
![image](https://user-images.githubusercontent.com/5714543/88486165-74f10d80-cf41-11ea-9fb9-74fb54e0539d.png)